kubernetes/apps_kubeclient.go

```diff
@@ -501,13 +502,13 @@ func (kc *kubeClient) getEnvironmentsFromConfigMap() (map[string]string, error)
 			if strings.HasPrefix(line, namespaceProp) {
 				tokens := strings.SplitN(line, ":", 2)
 				if len(tokens) < 2 {
-					return nil, errors.New("Malformed environments config map")
+					return nil, errs.New("malformed environments config map")
 				}
 				namespace = strings.TrimSpace(tokens[1])
 			}
 		}
 		if len(namespace) == 0 {
-			return nil, errors.New("No namespace for environment " + key + " in config map")
+			return nil, errs.New("no namespace for environment " + key + " in config map")
```

Comment:
Please use `errs.Errorf("no namespace for environment \"%s\" in config map", key)` instead.

Reply:
Done (same question)